### ISSUE TITLE
Take a look at codemod-cli

### ISSUE THREAD

**Issue by danielo515** [CodeshiftCommunity/CodeshiftCommunity#25]
Scanning npm for packages about codemods I have found codemod-cli:
https://www.npmjs.com/package/codemod-cli

It looks like a tool to create and manage collections of codemods. It takes care of:

- creating a new project, which includes a CLI to easily execute the codemods as a standalone CLI tool
- adding new codemods to the project, having an specific folder structure (that I find interesting)
- testing the codemods
- updating the documentation. Not sure how it does it, but I am pretty sure it picks the input and output fixtures to generate the examples

Maybe you should consider contacting the author and talk him about the codeshift community, or at least add it to the resources section?

**Comment by danieldelcore** [comment]
This is great, this is actually exactly what I'm hoping to do with the CLI! 

Most of that stuff is already built and working for codemods in the community folder, but not yet available for use externally via the CLI. That's what i'm planning to do next!

**Comment by danielo515** [comment]
One that I find particularly interesting is the documentation generation.
Many codemod repositories lack good documentation (specially examples) so
having them automatically is a big plus. If you don't have it planned I can
take a look

**Comment by danieldelcore** [comment]
Yeah absolutely, I actually made a bit of a start on this by generating some simple docs for the community folder here: https://www.codeshiftcommunity.com/docs/atlaskit__avatar

I wanted to extend that with a script that parses the tests/fixtures and renders them on the docs page as well. 
Not sure how that would look external to the repo, but it's definitely something worth investing in!
